packages/sdks/package.json

```diff
@@ -39,7 +39,7 @@
     "upgrade-example:all": "bash ./scripts/loop-command.sh upgrade-example \"svelte react-native solid qwik react vue\""
   },
   "dependencies": {
-    "@builder.io/mitosis": "^0.0.62",
+    "@builder.io/mitosis": "^0.0.63",
     "@builder.io/mitosis-cli": "^0.0.23",
```

Comment:
you need to update the CLI, since that has a dependency on mitosis and is still pointing to the older version I believe

Reply:
Opps, thanks for the tip